DataFormats/Common/interface/RangeMap.h

```diff
@@ -77,7 +82,9 @@ namespace edm {
     /// to use only comparators provided with CMSSW release.
     template<typename CMP> 
     range get(ID id, CMP comparator) const {
+#ifndef __APPLE__
       using namespace __gnu_cxx;
+#endif
```

Comment:
please check if this compiles without ```using namespace __gnu_cxx ```
unless you know offhand that it is needed for something here